website/docs/configuration/package-configurations.md

```diff
@@ -82,5 +82,5 @@ cli/build/install/ort/bin/ort report
   -o [reporter-output-dir]
   --report-formats PlainTextTemplate,WebApp
   --license-classifications-file $ORT_CONFIG_DIR/license-classifications.yml
-  --package-configuration-dir $ORT_CONFIG_DIR/packages
+  --package-configurations-dir $ORT_CONFIG_DIR/packages
```

Comment:
commit-message: The prefix should be: `docs(website): `

I believe a title like ``docs(website): Fix a typo`  with empty body should be fine.

Reply:
sure. I'll fix that

Reply:
done